packages/headless/src/composables/dataGrid/createDataGrid.testing.ts

```diff
@@ -0,0 +1,170 @@
+import { expect } from "@playwright/experimental-ct-vue";
+import type { Locator, Page } from "@playwright/test";
+import { createFocusCatcher } from "../../utils/playwright.js";
+
+const expectLabel = async (page: Page, grid: Locator) => {
+  const [labelledBy, label] = await Promise.all([
+    grid.getAttribute("aria-labelledby"),
+    grid.getAttribute("aria-label"),
+  ]);
+
+  if (labelledBy) {
+    expect(
+      label,
+      `"aria-labelledby" and "aria-label" must not both be defined on the grid element!`,
+    ).toBeFalsy();
+    expect(
+      page.locator(`#${labelledBy}`),
+      `"aria-labelledby" must point to an existing element!`,
+    ).toBeAttached();
```

Comment:
```suggestion
    ).toBeVisible();
```

Shouldn't it also be visible to the user?